src/config/crd/bases/goharbor.goharbor.io_assessmentreports.yaml

```diff
@@ -140,6 +140,25 @@ spec:
                         description: Generate indicates whether generate the assessment
                           report or not.
                         type: boolean
+                      governor:
+                        description: Indicate whether to config of governor
+                        properties:
+                          apiToken:
+                            description: Api token for user authentication
+                            type: string
+                          clusterId:
+                            description: Unique identifier of the cluster
```

Comment:
Also, update the governor properties in e2e tests.
/src/test/e2e/inspectionpolicyinspectionpolicy.yaml

Reply:
Check how to run e2e test.